aries-site/src/pages/components/grid.mdx

```diff
@@ -13,3 +24,245 @@ import { GridExample } from '../../examples';
+
+See `auto` and `flex` in action in the example below by adjusting your browser width, or applied in a [dashboard template](/templates/dashboards#two-column-dashboard).
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/MinMaxColumns2.js"
+  width={{ min: 'medium' }}
+  height="auto"
+  showResponsiveControls={['fullscreen']}
+  overflow="auto"
+>
+  <MinMaxColumns2 />
+</Example>
+
+### Rows
+Rows generally share the same properties as [columns](#columns). Accordingly, they won't be repeated here. 
+
+That said, there a two primary items to call attention to:
+1. Just like columns, rows are specified in an array. Each item in the array represent an additional row.
+1. The vast majority of time rows will be specified using a fixed height, or as `auto` to let the row's height be defined by its 
+contents.
+
+### Gap
+
+Use `gap` to define the spacing applied between columns and rows.
+
+- As best practice, use [t-shirt sizing](/foundation/tshirt-sizing?q=gap#t-shirt-sizing-for-spacing-and-other-styles) 
+to define gaps. T-shirt sizing keeps layouts consistent and easy to maintain.
+- Gap may be applied to columns and rows, independently or simultaneously.
+
+```
+// both column gaps and rows gaps will be medium
+gap="medium" 
+
+// column and row gaps applied independently
+gap={{ 
+  column: 'medium', 
+  row: 'xsmall'
+}}
+
+```
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/GapExample.js"
+  width={{ min: 'medium' }}
+  height="auto"
+>
+  <GapExample />
+</Example>
+
+## Implementing Grid
+Grid is not limited to the following implementations. Approaches may be mixed and matched
+as desired, but are presented with the intent to focus on the key concepts for each.
+
+### Fluid Grids
+Fluid grids allow the grid to grow and adapt to the grid's container width and the content provided. 
+
+As more content is added, it fills the Grid left to right, top to bottom. Columns are added or removed as 
+the screen's width is adjusted.
+
+- Great for presenting dynamic content where the number of items in the grid are unknown. Examples might 
+be applying filters to a catalog of services, presenting a dashboard where the user may customize the 
+included widgets, or image results returned from a search query.
+- Make responsive layouts easy to implement.
+- Often implemented with [uniform columns](#uniform-columns).
+
+The following example demonstrates a fluid grid consisting of 'small' columns. Adjust your browser's width 
+and notice how the grid adjusts by adding/removing columns and rows.
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/FluidGrid1.js"
+  width="100%"
+  height="auto"
+>
+  <FluidGrid1 />
+</Example>
+
+<br />
```

Comment:
We should probably consider how to make "Example" smarter in creating space below itself when it isn't followed by a heading, but that is a separate PR/scope of work.

Reply:
I'm good with this for the time being.